java/create-and-save-map/README.md

```diff
@@ -10,13 +10,15 @@ Maps can be created programmatically in code and then serialized and saved as an
 
 ## How to use the sample
 
-Use the navigation drawer to select the basemap and layers you'd like to add to your map. Tap the save button and enter a title, tags, and description for your map and then hit 'Save Map'. Sign into an ArcGIS Online account when prompted and save the map.
+When you run the sample, you will be challenged for an ArcGIS Online login. Enter a user name and password for an ArcGIS Online named user account (such as your ArcGIS for Developers account). Then, choose the basemap and layers for your new map. To save the map, add a title, tags, description (optional), and a folder on your portal (you will need to create one in your portal's My Content section if you don't already have one). Click the Save button to save the map to the chosen folder.
 
 ## How it works
 
-1. An `ArcGISMap` is created with a `Basemap` and a few operational layers.
-2. A `Portal` object is created and loaded. This will issue an authentication challenge, prompting the user to provide credentials.
-3. Use `ArcGISMap.saveAsAsync(...)` passing in the `Portal`, and information about the map, to save the map to the portal.
+1. Set a `DefaultAuthenticationChallengeHandler` to the app's `AuthenticationManager`.
+2. Create a new `Portal` and load it. Once it has loaded, an authentication challenge will be issued and you'll be prompted to enter your username and password.
```

Comment:
another tweak from the design (changed since this was more user focussed than dev focussed) :D 

```suggestion
2. Create a new `Portal` and load it to invoke the authentication challenge.
```